Base/res/ladybird/newtab.html

```diff
@@ -89,6 +90,10 @@
             } else if (searchEngine == "yandex") {
                 url = new URL("https://yandex.com/search");
                 url.searchParams.set("text", query);
+            } else if (searchEngine == "wikipedia") {
+                url = new URL("https://en.wikipedia.org/w/index.php?title=Special:Search&search=YOUR_SEARCH_TERM");
+                url.searchParams.set("title", "Special:Search");
+                url.searchParams.set("search", query);
```

Comment:
Why are we setting search params in the URL itself and then again right after? Seems like this can just be:
```js
url = new URL("https://en.wikipedia.org/w/index.php?title=Special:Search");
url.searchParams.set("search", query);
```

Reply:
Oh, how did this get here 0_0 I think this all happened while I was trying to fixup my commits! I haven't used Git to contribute to another repository before.